docs/src/modules/components/AppDrawer.js

```diff
@@ -19,7 +19,7 @@ const styles = theme => ({
   },
   title: {
     color: theme.palette.text.secondary,
-    marginBottom: theme.spacing.unit / 2,
+    marginBottom: 4,
```

Comment:
`theme.spacing(1) / 2`?

(Or should this not have been using theme.spacing in the first place?)

Reply:
That is also something that came up in https://github.com/mui-org/material-ui/issues/13663#issuecomment-458771447

I don't think the spacing is appropriate here. Maybe change the base spacing in the theme and verify that this is still producing a "valid" layout.

Reply:
Are you saying 4px is the wrong spacing, or that using `theme.spacing(0.5)` is incorrect? 

If it's the latter, there's at least one other occurrence that would need to be changed.

Reply:
I don't know if it's incorrect. Just looking at this from through the lens of semantics. The UI should align to the grid that is created by the base spacing of `x` px. If we recognize that it is useful to circumvent that then we shouldn't warn about it in the `Grid` component.

I would hope that I could "just" change the base spacing and the UI still looks good. If it breaks in this component then it is a pretty good indicator that the spacing is used incorrectly.